conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -366,26 +348,21 @@ private boolean isByteBuffer(Type type) {
         Type type = enriched.conjureDef().getType();
 
         if (type.accept(TypeVisitor.IS_LIST)) {
-            return ImmutableList.of(
-                    createCollectionSetter("addAll", enriched),
-                    createItemSetter(enriched, type.accept(TypeVisitor.LIST).getItemType()));
```

Comment:
@dansanduleac I think the number of parens in `createItemSetter(enriched, type.accept(TypeVisitor.LIST).getItemType())` makes it a bit subtle to read when it's re-flowed in the new 'dense' style.  Thoughts on having a heuristic that would value having opening and closing pairs of parens on the same line?